engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/text/AnyToDisplayTextNode.java

```diff
@@ -32,6 +35,22 @@ Text showExceptions(
     }
   }
 
+  @Specialization
+  Text convertText(Text self) {
+    final var limit = 80;
```

Comment:
shouldn't that be lifted to a (`private static final`) constant ?

Reply:
+1

Reply:
Originally the `limit` was only used locally in a single method. Then I had to move the iterator manipulation behind `@TruffleBoundary`...